sdks/java/core/src/main/java/org/apache/beam/sdk/util/state/StateContexts.java

```diff
@@ -24,7 +24,7 @@
 import org.apache.beam.sdk.values.PCollectionView;
 
 /**
- * Factory that produces {@link StateContext} based on different inputs.
+ * Static methods for working with {@link StateContext}.
```

Comment:
Just future-proofing the javadoc. In a perfect world _class_`s` would simply not require Javadoc. Scala got it right. I'll copy in the old description I suppose.